refinery/data_set_manager/search_indexes.py

```diff
@@ -153,6 +154,7 @@ def prepare(self, node):
 
         filetype = "" if file_store_item is None else file_store_item.filetype
         data.update({
+            NodeIndex.DATAFILE: node.get_file_store_item().datafile,
```

Comment:
I think this is troublesome for many `Nodes`. The majority of `Nodes` don't have `FileStoreItems`, which would yield `AttributeErrors` when trying to access `.datafile` on a `None`.

I think adding a few tests to `data_set_manager.tests.NodeIndexTests` regarding what `NodeIndex.DATAFILE` should contain based on at least the two scenarios (`Node` has/doesn't have a `FileStoreItem`) would be a good idea